src/Render/Markdown/TemplateRenderer.cs

```diff
@@ -1,10 +1,21 @@
 ﻿using Scriban;
+using System.IO;
+using System.Reflection;
 using System.Threading.Tasks;
 
 namespace D2L.Dev.Docs.Render.Markdown {
 	internal class TemplateRenderer {
 		private readonly Template m_template;
-		
+
+		public static TemplateRenderer Create( string resourceName ) {
+			resourceName = resourceName.Replace( "/", "." );

```

Comment:
Since the argument is a resource name, maybe don't do this? i.e. the caller knows it's a resource